deps/wazuh_testing/wazuh_testing/modules/aws/data_generator.py

```diff
@@ -155,10 +155,79 @@ def get_data_sample(self) -> str:
         return buffer.getvalue()
 
 
+class ConfigDataGenerator(DataGenerator):
+    BASE_PATH = join(cons.AWS_LOGS, cons.RANDOM_ACCOUNT_ID, cons.CONFIG, cons.US_EAST_1_REGION)
+    BASE_FILE_NAME = f"{cons.RANDOM_ACCOUNT_ID}_{cons.CONFIG}_{cons.US_EAST_1_REGION}_ConfigHistory_AWS_"
+
+    def get_filename(self, prefix=None, **kwargs) -> str:
+        """Return the filename in the Config format.
+
+        Example:
+            <prefix>/AWSLogs/<suffix>/<organization_id>/<account_id>/Config/<region>/<year>/<month>/<day>
+
+        Returns:
+            str: Syntetic filename.
+        """
+        now = datetime.now()
```

Comment:
I wonder if we should use datetime.utcnow() to avoid inconsistencies related to timezone. 

What is the exact purpose of this? If we can use utcnow, please change it (in all appearances where it is used and possible to do so related to these tests and other PRs).